### ISSUE TITLE
SA0102 False positive

### ISSUE THREAD

**Issue by elyoh** [Visual-Stylecop/Visual-StyleCop#39]
The following line:
```c#
var x = y ? z?.Invoke() : w?.Invoke();
```
incorrectly gives:
SA0102 : CSharp.CsParser : A syntax error has been discovered in file ....

Upon further investigation, it is the first use of the null conditional operator 
```c#
z?.Invoke() 
```
which appears to causes the SA0102 error.

Build 4.7.58.3

**Comment by ptittof57** [comment]
Fixed in the next release
